File: lab_3/src/prime.c
#include <stdio.h>
int main()
{
   int i = 1,n,r,factor=0;
   	printf("Enter a number: ");
   	scanf("%d",&n);
   	while(i<=n)
   	{
   	   r = n%i;
   	   if(r==0)
   	   {
   	   	factor++;
   	   	i++;
   	   }
   	   else
   	   i++;
		  	
   	}
   	if(factor==2)
   	{
   		printf("The number you have entered is a prime number = %d",n);
   	}
   	else
   	printf("The number you have entered is not a prime number = %d",n);
	
	
	
	
}

	  
    



	  	
	  

File: src/prime.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */

/**
 *
 * @author shital
 */
import java.util.Scanner;
public class prime {
    public static void main(String args[])
    {
        System.out.println("Please enter a number: ");
        Scanner sc = new Scanner(System.in);
        int pri = sc.nextInt();
        
        int i = 1;
       int factor = 0;
        if(i<=pri)
        {
             int r;
            r = pri%i;
            
            if(r==0)
            {
                
                 factor++; 
               i++;
            }
            else
               
                i++;
            if(factor==2)
        {
          System.out.println("The number you have entered is a prime number ");  
        }
        else
            System.out.println("The number you have entered is not a prime number ");
        }
        
    }
}
